src/backend/app/main.py

```diff
@@ -17,13 +29,41 @@
     allow_headers=["*"],
 )
 
+@app.get("/health")
+async def health_check():
+    print(os.getenv("DATABASE_URL"))
+    if crud.check_db_connection():
+        return {"status": "healthy"}
+    else:
+        return {"status": "unhealthy"}
+
+@app.get("/echo_env/")
+async def echo_env():
+    return {"url": os.environ.get("DATABASE_URL", "ERROR"), "env": dict(os.environ)}
 
 @app.post("/generate_prediction/")
 async def generate_prediction(request: Request):
     response = tools.generate_prediction(await request.json())
     return JSONResponse(content=response)
 
 @app.post("/save_annotation/")
-async def save_annotation(request: Request):
-    response = tools.save_annotation(await request.json())
```

Comment:
if we no longer use some functions, then they can be removed (in separate commits, of course)